llms/gguf_llm/models.py

```diff
@@ -18,6 +18,7 @@ class ModelArgs:
     num_attention_heads: int
     rms_norm_eps: float
     vocab_size: int
+    context_length: int
```

Comment:
Do the other models in the list have this field?

Reply:
Yes they all do with no errors during generation.

<img width="795" alt="Screenshot 2024-04-25 at 12 28 27" src="https://github.com/ml-explore/mlx-examples/assets/11355002/4b4b9cae-907c-4801-bb29-d03345b373c4">

Reply:
Is everything okay for a merge now? @awni

Reply:
Yea I think so. I was wondering about a nit: if we wanted to print all the model info by default.

Reply:
Since the use-case here is primarily for research I believe having to see relevant model dets is essential, also it's not all the model info, just some relevant ones. I learned that from karpathy's gpt repos:)

Reply:
Interestingly, the official [Phi-3-mini-4k-instruct-gguf](https://huggingface.co/microsoft/Phi-3-mini-4k-instruct-gguf)'s is using llama architecture, you can see even in the metadata we are printing, prints out because of that.

<img width="1309" alt="Screenshot 2024-04-27 at 07 30 50" src="https://github.com/ml-explore/mlx-examples/assets/11355002/bfff9a46-55fd-4902-9714-82e5a5548eb4">

Reply:
> Yea I think so. I was wondering about a nit: if we wanted to print all the model info by default.

Interestingly, the official [Phi-3-mini-4k-instruct-gguf](https://huggingface.co/microsoft/Phi-3-mini-4k-instruct-gguf) is using llama architecture, you can see even in the metadata we are printing, prints out because of that.

<img width="1309" alt="Screenshot 2024-04-27 at 07 30 50" src="https://github.com/ml-explore/mlx-examples/assets/11355002/bfff9a46-55fd-4902-9714-82e5a5548eb4">

Reply:
@awni still no merge?